src/RA_StringUtils.h

```diff
@@ -309,8 +309,8 @@ class StringBuilder
         }
         else
         {
-            const int nLength = strlen(arg);
-            int nPadding = std::stoi(sFormat.c_str());
+            const auto nLength = std::strlen(arg);
+            std::ptrdiff_t nPadding{std::stoi(sFormat)};
```

Comment:
another data type? why not `size_t`?

Reply:
because `stoi` returns int but a narrowing error might happen below it if it's not 8 bytes.